content/release-10-18-0/digital-twin-manager-10-18-0-bundle/10-18-0-1.md

```diff
@@ -0,0 +1,355 @@
+<td>CTM-429</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now import or export a maximum of 250 asset models and 500 asset properties.</td>
+<td>CTM-454</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now move the child assets to another asset which belongs to the same asset model.</td>
+<td>CTM-491</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A breadcrumb is shown for the <b>Asset model samples</b> page.</td>
+<td>CTM-563</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset tree</b> tab, the count for the number of asset models has been removed.</td>
+<td>CTM-559</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset tree</b> tab, a <b>Reload</b> button is now displayed.</td>
+<td>CTM-555</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The import assets layout has been modified to be in sync with other layouts.</td>
+<td>CTM-550</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Localizations have been added for the <b>Subassets</b> page.</td>
+<td>CTM-373</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Cell renderers have been added for child asset models and asset properties columns in the <b>Export Asset models</b> popup.</td>
+<td>CTM-583</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset model</b> creation page, the <b>New asset property</b> button is always enabled and is by default assigned to the asset model after creation.</td>
+<td>CTM-309</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset model or property gets modified.</td>
+<td>CTM-321</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset model is deleted from the backend.</td>
+<td>CTM-333</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A warning message is shown in the <b>Subassets</b> page when the associated asset property is deleted from the backend.</td>
+<td>CTM-338</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>DTM application name changed from <b>Digital Twin Manager</b> to <b>digital-twin-manager</b> in order to enable it as default subscribed application in Cumulocity.</td>
+<td>CTM-504</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A set of pre-defined Asset models are added under <b>Asset model samples</b> page which will make much easier for users to understand the concept of Asset models and starting to work with it.</td>
+<td>CTM-483</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td><b>Subassets</b> page is implemented using plugin approach.</td>
+<td>CTM-207</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Assets move feature is applied with required CSS changes.</td>
```

Comment:
@spau6 where can you find this feature in the UI?

Reply:
One has to select assets using the selection box  in the Assests page, then this option/Move will appear

Reply:
@spau6 Is this a button, an icon or another type of UI feature?

Reply:
It is "Move selected" button, "help" button